influxql/ast_test.go

```diff
@@ -295,6 +295,7 @@ func TestSelectStatement_HasWildcard(t *testing.T) {
 
 	for i, tt := range tests {
 		// Parse statement.
+		t.Logf("statement: %s", tt.stmt)
```

Comment:
If we're going to have this in the test output, can we at least at the testname and index, so the output has some context? Otherwise it just adds a lot of next-to useless noise to the test output. (IMHO).